contracts/delegatecall/CallToENS.sol

```diff
@@ -0,0 +1,90 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.20;
+
+import {ENS} from "@ensdomains/ens-contracts/contracts/registry/ENS.sol";
+import {AddrResolver} from "@ensdomains/ens-contracts/contracts/resolvers/profiles/AddrResolver.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {IERC721} from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import {IERC1155} from "@openzeppelin/contracts/token/ERC1155/IERC1155.sol";
+
+import {LibError} from "../library/LibError.sol";
+
+contract CallToENS {
+    using LibError for bytes;
+    using SafeERC20 for IERC20;
+
+    ENS immutable _ens;
+    constructor(ENS ens) {
+        _ens = ens;
+    }
+
+    function sendCallToENS(
+        bytes32 node,
+        uint256 value,
+        bytes memory payload
+    ) external {
+        address recipient = resolveENS(node);
+
+        (bool success, bytes memory returndata) = recipient.call{value: value}(
+            payload
+        );
```

Comment:
You can first declare these variables, and then assign them, so that the line fits in 80 characters.

Reply:
It's not more than 80 chars in its current way actually

Reply:
It is 89 chars long, that's why `prettier` is breaking this line.
My suggestion is the following.

```suggestion
        bool success;
        bytes memory returndata;

        (success, returndata) = recipient.call{value: value}(payload);
```